examples/eSealing/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "e_sealing",
     "packageManager": "yarn@3.2.0",
-    "version": "1.1.2",
+    "version": "1.1.3",
     "license": "Apache-2.0",
     "dependencies": {
-        "@concordium/react-components": "^0.3.0",
-        "@concordium/web-sdk": "^6.3.0",
+        "@concordium/react-components": "0.4.0-rc.5",
```

Comment:
Well `@concordium/react-components` depend on `@concordium/browser-wallet-api-helpers`, which is also released currently as RC, and should probably be until released alongside the browser wallet to avoid people installing unless specifically doing `@concordium/browser-wallet-api-helpers@beta`.

Do you have another solution that would be preferable?

Reply:
No, I think that is fine, but then I also expect there to be a task to ensure that all the correct versions are released and then update accordingly when we get to that point.

Reply:
There are tasks to update to officially released versions in both this repo and that of dapp libraries (which holds the react-components package).